Makefile

```diff
@@ -42,82 +43,16 @@ else
   TMAP8_DIR        ?= $(shell dirname `pwd`)/tmap8
 endif
 
-# Use the Cardinal submodule if it exists and CARDINAL_DIR is not set
-# If it doesn't exist, and CARDINAL_DIR is not set, then look for it adjacent to the application
-CARDINAL_SUBMODULE    := $(CURDIR)/cardinal
-ifneq ($(wildcard $(CARDINAL_SUBMODULE)/Makefile),)
-  CARDINAL_DIR        ?= $(CARDINAL_SUBMODULE)
-else
-  CARDINAL_DIR        ?= $(shell dirname `pwd`)/cardinal
-endif
-
-# Cardinal contrib variables
-CARDINAL_CONTRIB_DIR           := $(CARDINAL_DIR)/contrib
-CARDINAL_CONTRIB_INSTALL_DIR   ?= $(CARDINAL_DIR)/install
-OPENMC_INSTALL_DIR             := $(CARDINAL_CONTRIB_INSTALL_DIR)
-OPENMC_LIBDIR                  := $(OPENMC_INSTALL_DIR)/lib
-OPENMC_BUILDDIR                := $(CARDINAL_DIR)/build/openmc
-OPENMC_INSTALL_DIR             := $(CARDINAL_CONTRIB_INSTALL_DIR)
-OPENMC_INCLUDES                := -I$(OPENMC_INSTALL_DIR)/include
-OPENMC_LIBDIR                  := $(OPENMC_INSTALL_DIR)/lib
-OPENMC_LIB                     := $(OPENMC_LIBDIR)/libopenmc.so
-
-PETSC_DIR           ?= $(MOOSE_DIR)/petsc
-PETSC_ARCH          ?= arch-moose
-LIBMESH_DIR         ?= $(MOOSE_DIR)/libmesh/installed/
-
-# This is the Eigen3 location on CIVET. If you are using MOOSE's conda environment,
-# you don't need to set these variables, because conda sets them for you. The only
-# scenario where you might need to manually set these is if you're not using the
-# conda environment. You will get a compile error about FindEigen3.cmake if you
-# do indeed need to set these.
-EIGEN3_DIR          ?= $(LIBMESH_DIR)/include
-Eigen3_DIR          ?= $(EIGEN3_DIR)
-
-# If HDF5_DIR is set, use those settings to link HDF5 to OpenMC.
-# Otherwise, use the PETSc location, as the self-built HDF5 libraries will be housed within.
-ifeq ($(HDF5_DIR),)
-  HDF5_DIR          := $(PETSC_DIR)
-  export HDF5_DIR
-endif
-
-# HDF5_ROOT is used in makefiles for the Cardinal third-party dependencies.
-# Set it using HDF5_DIR
-HDF5_ROOT          := $(HDF5_DIR)
-export HDF5_ROOT
+# Check for optional dependencies and, if found, configure for building.
+# Set default for ENABLE_CARDINAL.
+ENABLE_CARDINAL := no
```

Comment:
Doesn't this define the default as NOT having CARDINAL? 

I think the default should be to include cardinal.

Reply:
Addressed in 93cb186.